mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt

```diff
@@ -184,11 +197,9 @@ class ITMGeolocationManager(private var context: Context) {
         }
 
     private var _fusedLocationClient: FusedLocationProviderClient? = null
-    private val fusedLocationClient: FusedLocationProviderClient
-        get() {
-            val client = _fusedLocationClient ?: getFusedLocationProviderClient()
-            _fusedLocationClient = client
-            return client
+    private val fusedLocationClient
+        get() = _fusedLocationClient ?: getFusedLocationProviderClient().also {
+            _fusedLocationClient = it

```

Comment:
I _think_ we can get rid of the private backing var and just do this:
```suggestion
    private val fusedLocationClient by lazy { getFusedLocationProviderClient() }
```

Reply:
I pushed an update to use lazy here and one other place.